{
    "config": {
        "error": {
            "cannot_connect": "Ha fallat la connexi\u00f3",
            "invalid_access_token": "Token d'acc\u00e9s inv\u00e0lid",
            "unknown": "Error inesperat"
        },
        "step": {
            "user": {
                "data": {
                    "access_token": "Token d'acc\u00e9s"
                }
            }
        }
    },
    "entity": {
        "climate": {
            "driver_temp": {
                "name": "Climatitzaci\u00f3"
            }
        }
    }
}